### ISSUE TITLE
ModelCheckpoint does not create checkpoint files

### ISSUE THREAD

**Issue by aprbw** [PyTorchLightning/pytorch-lightning#9228]
## 🐛 Bug

ModelCheckpoint does not create checkpoint files

### To Reproduce

https://colab.research.google.com/drive/137Q582QEjECbDR8iAF4bZASM8a3i9TG1?usp=sharing

### Expected behavior

When run, a .ckpt files to be created under ./lightning_logs/e00_default_sweep/default_run/checkpoints/

### Environment

* CUDA:
	- GPU:
		- Tesla T4
	- available:         True
	- version:           10.2
* Packages:
	- numpy:             1.19.5
	- pyTorch_debug:     False
	- pyTorch_version:   1.9.0+cu102
	- pytorch-lightning: 1.4.4
	- tqdm:              4.62.0
* System:
	- OS:                Linux
	- architecture:
		- 64bit
		- 
	- processor:         x86_64
	- python:            3.7.11
	- version:           #1 SMP Sat Jun 5 09:50:34 PDT 2021

**Comment by rohitgr7** [comment]
![Screenshot from 2021-09-01 01-12-25](https://user-images.githubusercontent.com/30778939/131567103-8c11c08b-fa3d-473a-907c-767064ef1e22.png)
working fine for me.

**Comment by aprbw** [comment]
![image](https://user-images.githubusercontent.com/42436117/131629872-88823986-f0da-4121-9626-62ff1f64901a.png)

Okay, thanks a lot!
It is actually working, but something is wrong with colab it seems.
Usually all the files created appears in the files explorer tab at the left
like downloaded data, profiler and tensorboard (I tried it in another notebook)
But for some reason, the .ckpt files are not appearing. (maybe for formatting reasons, even after waiting for a long time and refreshing)

**Comment by aprbw** [comment]
![image](https://user-images.githubusercontent.com/42436117/131665677-b86aef14-de71-406e-9b1c-800ce3d6f3b8.png)

Just wanna put this as a known issue, just in case someone else is wondering as well.

So I there is a bug, but not with PL, maybe with colab.

All the other files appears: tensorboard, profiler, hparams.yaml (note, this is not the posted minimal example).

But the .ckpt files are not appearing for some reasons. I have renamed the file, so it is not issues with special characters "=".

I have refreshed the colab file explorer after waiting a while. The other files appeared, but not the .ckpt.
(note the checkpoints folder is opened (arrow is down))
